docs/src/pages/components/toggle-button/ToggleButtonSizes.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import ToggleButton from '@material-ui/lab/ToggleButton';
+
+const useStyles = makeStyles(theme => ({
+  margin: {
+    margin: theme.spacing(1),
+  },
+}));
+
+function ButtonSizes() {
+  const classes = useStyles();
+
+  return (
+    <div>
+      <div>
+        <ToggleButton size="small" value="small" className={classes.margin}>
```

Comment:
Thanks for adding a demo, but it doesn't portray the ToggleButton particularly well. Perhaps three different sizes of a group? (Either discrete examples, or dynamically changing the size.)

Reply:
I changed the demo to a ToggleButtonGroup whose size can be dynamically changed. Let me know what you think!

Reply:
Looks good!